gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -262,3 +258,30 @@ def validate(self, parsed_args):
             '--variant_merge_strategy {}|{}'.format(
                 MergeOptions.MOVE_TO_CALLS,
                 MergeOptions.MERGE_WITH_NON_VARIANTS))
+
+
+class PreprocessOptions(VariantTransformsOptions):
+  """Options for preprocess."""
+  _REPORT_NAME = 'conflicts_report.csv'
+  _RESOLVED_HEADERS_FILE_NAME = 'resolved_headers.vcf'
+
+  def add_arguments(self, parser):
+    parser.add_argument(
+        '--report_all',
+        type='bool', default=True, nargs='?', const=False,
```

Comment:
Should use `const=True` (this is basically saying that just using `--report_all` results to true).